py/torch_tensorrt/dynamo/backend/lowering/_partition.py

```diff
@@ -14,6 +15,11 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_SINGLE_NODE_PARTITIONS: Set[str] = set(
+    _get_qualified_name(module.new_operator)
+    for module in MODULE_SUBSTITUTION_REGISTRY.values()
+)
```

Comment:
Updated registry to use strings formatted in a streamlined/uniform way via [`get_qualified_name`](https://github.com/pytorch/pytorch/blob/17166c2511fa8b18378438b4283f2d2b4625356e/torch/fx/node.py#L79) as is already done in partitioning, and addressing the issues raised here: https://github.com/pytorch/TensorRT/pull/1921#discussion_r1212225354